hebiros_advanced_examples/launch/mapping_navigation/maps/Hebi_map.yaml

```diff
@@ -0,0 +1,8 @@
+#image: /home/edward/Hebi_map.pgm
```

Comment:
Not a bad idea to have a map example, but I wonder about the usefulness of having this map in the repo?  Maybe have a link in the README to what an example map might look like instead?

It's not a huge file, so I don't feel strongly on this.